demos/index.html

```diff
@@ -48,6 +48,12 @@ <h2>Morningstar Connectors Demos</h2>
                 <div class="sidebar-navigation">
                     <h3>Demos</h3>
                     <ul>
+                        <li>
+                            <button onclick="selectDemo('DWS-AssetAllocBreakdown/demo.html', this)">
```

Comment:
I think that we should stick with the following notation: `dws-asset-alloc-breakdown`.